Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Microsoft365Defender_description.md

```diff
@@ -22,16 +22,21 @@ In order to use the Cortex XSOAR application, use the default application ID.
 To use a self-configured Azure application, you need to add a new Azure App Registration in the Azure Portal, with mobile and desktop flows enabled.
 
 #### Required Permissions
-* microsoft-365-defender-incidents-list (one of the following):
-    * Incident.Read.All	- Application
-    * AdvancedHunting.Read.All - Application
+* microsoft-365-defender-incidents-list:
```

Comment:
no need to split into commands here, just a list of all permissions

Reply:
Done